src/main/java/com/valmiki/adhyapakservice/exception/DataNotFoundException.java

```diff
@@ -0,0 +1,15 @@
+package com.valmiki.adhyapakservice.exception;
+
+import org.springframework.http.HttpStatus;
+
+public class DataNotFoundException extends Exception {
+    //TODO: make custom exception
+    private ErrorResponse errorResponse;
+
+    public DataNotFoundException() {
+        super("Data not found");
```

Comment:
Don't use literal, create a final variable above

Reply:
removed this part do i still need to create a final variable

Reply:
Yes, add a final string variable above and use that instead of literal

Reply:
fixed